neuromation/cli/job.py

```diff
@@ -1127,3 +1169,48 @@ def calc_statuses(status: Sequence[str], all: bool) -> Set[JobStatus]:
                 return parse_columns(format_str)
         return COLUMNS
     return format
+
+
+async def calc_life_span(client: Client, value: Optional[str]) -> Optional[float]:
+    delta = (
+        _parse_timedelta(value)
+        if value is not None
+        else await calc_default_life_span(client)
+    )
+    seconds = delta.total_seconds()
+    if seconds == 0:
+        return None
+    assert seconds > 0, f"life-span cannot be negative, got: {seconds}"
+    return seconds
+
+
+async def calc_default_life_span(client: Client) -> timedelta:
+    config = await client.config.get_user_config()
+    section = config.get("job")
+    life_span = DEFAULT_JOB_LIFE_SPAN
+    if section is not None:
+        value = section.get("life-span")
```

Comment:
Please document this parameter in `neuro help user-config`

Reply:
wow, didn't know about this command. OK